cmd/neofs-node/netmap.go

```diff
@@ -81,8 +81,13 @@ func (s *networkState) setNodeInfo(ni *netmapSDK.NodeInfo) {
 	s.controlNetStatus.Store(ctrlNetSt)
 }
 
-func (s *networkState) controlNetmapStatus() control.NetmapStatus {
-	return s.controlNetStatus.Load().(control.NetmapStatus)
+func (s *networkState) controlNetmapStatus() (res control.NetmapStatus) {
+	v := s.controlNetStatus.Load()
+	if v != nil {
```

Comment:
Can we instead store `Undefined` or `OFFLINE` during initialization?

Reply:
yes, that looks better, changed